source/material_model/rheology/strain_dependent.cc

```diff
@@ -147,6 +147,24 @@ namespace aspect
                            "for background material and compositional fields, "
                            "for a total of N+1 values, where N is the number of compositional fields. "
                            "If only one value is given, then all use the same value.  Units: None.");
+
+        prm.declare_entry ("Strain healing mechanism", "no healing",
+                           Patterns::Selection("none|temperature dependent"),
+                           "Whether to apply strain healing to viscosity, and if yes, which method to "
```

Comment:
The method is applied to strain terms for plastic yielding and not just viscosity, yes?